webapps-backup/src/main/java/io/camunda/webapps/backup/repository/opensearch/OpenSearchGetSnapshotResponse.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.backup.repository.opensearch;
+
+import java.util.List;
+import java.util.Objects;
+import org.opensearch.client.opensearch.snapshot.GetSnapshotResponse;
+
+public class OpenSearchGetSnapshotResponse {
```

Comment:
:wrench: Looks to me like it can be a record - only carries data, no real business logic.

Reply:
I just copy-pasted it from the original source without any changes, should I convert it to a record?